### PR TITLE
[DOCS] Exceptions flyout: update screenshots and procedure

### LINKED ISSUE #1468: [DOCS] Update exceptions modal screenshots
## Description 

[This PR](https://github.com/elastic/kibana/pull/123408) updates the exceptions UX to a flyout component instead of a modal and the related docs and screenshots will need to be updated to match the new UI.

We are currently planning on implementing this change in 8.1

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #1468.

Preview: [Rule exceptions and value lists](https://security-docs_1619.docs-preview.app.elstc.co/guide/en/security/master/detections-ui-exceptions.html)

### CODE REVIEW

**Review by benironside — APPROVED** [review]
Looks crisp and nutritious 👍

**Review by dplumlee — APPROVED** [review]

File: docs/detections/detections-ui-exceptions.asciidoc [hunk 1]
```diff
@@ -99,23 +99,24 @@ specific event in the sequence, update the rule's EQL statement. For example:
...
-. To add an exception via the Alerts table:
+* To add an exception from the Alerts table:
 .. Go to *Detect* -> *Alerts*.
-.. Scroll down to the Alerts table and click the *More Actions* button (*...*), then select *Add rule exception*.
-+
-[role="screenshot"]
-image::images/more-action-button.png[]
-+
-The *Add Rule Exception* window opens (via the Alerts table).
+.. Scroll down to the Alerts table, and from an alert for which you want to create an exception, click the *More Actions* button (*...*), then select *Add rule exception*.
```
Comment (nastasha-solomon) [inline docs/detections/detections-ui-exceptions.asciidoc:116]:
```suggestion
.. Scroll down to the Alerts table, go to the alert you want to create an exception for, click the *More Actions* button (*...*), then select *Add rule exception*.
```

Comment (nastasha-solomon) [reply docs/detections/detections-ui-exceptions.asciidoc:116]: Just a rec to simplify the middle part of this step.

**Review by nastasha-solomon — APPROVED** [review]
Left one minor comment, which you can take or leave - otherwise, LGTM!

**Review by jmikell821 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**